### PR TITLE
added new file mailroom.py

### PR DESCRIPTION
**Pull request by Chieu1**

### CODE REVIEW

**Comment by Chieu1** [comment]
Added file mailroom.py

File: students/chieu_quach/lesson03/mailroom.py [hunk 1]
```diff
@@ -0,0 +1,157 @@
...
+
+len_donation_list = len(donation_list)
+prompt = "\n".join(("Welcome to the Mailroom Part 1",
+                    "Please choose from below options: ",
+                    "1 - Send a Thank You",
+                    "2 - Create a Report",
+                    "3 - Quit",
+                    ">>>"))
+
+from operator import itemgetter
```
Comment (charlesrobison) [inline students/chieu_quach/lesson03/mailroom.py]: You'll want to move this to the beginning of your script, along with the other import statements

File: students/chieu_quach/lesson03/mailroom.py [hunk 2]
```diff
@@ -0,0 +1,157 @@
...
+
+def send_thankyou():
+
```
Comment (charlesrobison) [inline students/chieu_quach/lesson03/mailroom.py]: You can remove this blank row

File: students/chieu_quach/lesson03/mailroom.py [hunk 3]
```diff
@@ -0,0 +1,157 @@
...
+      name_found = ""
+      response = input(" Please enter full name ") 
+      for z in donation_list:
+            
+            if response == "list":
+            
+               if e == 1:
+                  print ("List of Names \n")
+               e = e + 1
+               print (z)
```
Comment (charlesrobison) [inline students/chieu_quach/lesson03/mailroom.py]: No space between print and (

File: students/chieu_quach/lesson03/mailroom.py [hunk 4]
```diff
@@ -0,0 +1,157 @@
...
+            elif response == z[0]:
+               name_found = "yes"
+               name = z
+            else:
+               continue
+      print ("\n")
+      # if donation name is located 
+      if name_found == "yes": 
+            amount_donate =  float(input(" Please Enter Donation Amount: "))
+
```
Comment (charlesrobison) [inline students/chieu_quach/lesson03/mailroom.py]: Remove these blank rows

**Comment by charlesrobison** [comment]
@Chieu1 - make sure you submit this for grading.  You'll want to submit the code for grading at the same time as you create your PRs - let me know if you have questions.

**Comment by Chieu1** [comment]
Charles,

Thanks for checking exercise mailroom.py.
Can you please follow up with these issues for me ?
I have submitted all 3 exercises from lesson 2 for grading. I logged in and checked Course progress and found they have not been graded yet.
I found the exercises in lesson 3, which I have not submiited for grading yet, get graded insread.
I think there is little mix up here.
I will submit 4 exercises in lesson 3 for grading soon.
I have a question on the lesson 4 exercise “Dictionary and set Lab”, 
On the Dictionaries 2, what it means “using the same keys but with the number of ‘t’s in each vlue.

Thanks,

Chieu

**Comment by natasha-aleksandrova** [comment]
@Chieu1 looks like you asked identical questions in the email - please wait for Charles reply so that we're not having the same conversations in multiple places.

**Comment by Chieu1** [comment]
Okay, Thanks Natasha

Chieu

**Pull request merged**